sklearn/tests/test_calibration.py

```diff
@@ -275,6 +275,34 @@ def multiclass_brier(y_true, proba_pred, n_classes):
     assert calibrated_brier < 1.1 * uncalibrated_brier
 
 
+def test_calibration_zero_probability():
+    class FakeClassifier():
+        def __init__(self, n_classes):
+            self.classes_ = n_classes
+
+        def predict_proba(self, X):
+            return np.empty_like(X)
+
+    class FakeCalibrator():
+        # This function is called from _CalibratedClassifier.predict_proba.
+        def predict(self, X):
+            return np.zeros_like(X)
+
+    X, y = make_blobs(n_samples=500, n_features=100, random_state=7,
+                      centers=10, cluster_std=15.0)
+    n_classes = np.unique(y)
+
+    clf = FakeClassifier(n_classes)
+    calibrator = FakeCalibrator()
+    cal_clf = _CalibratedClassifier(
+        base_estimator=clf, calibrators=[calibrator], classes=n_classes)
+
+    probas = cal_clf.predict_proba(X)
+
+    # Check all probabilities are 1. / n_classes
+    assert_allclose(probas, np.full_like(probas, 1. / len(n_classes)))
```

Comment:
assert_allclose uses broadcasting so assert_allclose(probas, 1. / len(n_classes)) works

Reply:
I did not know about that. I removed redundant parts.